python/Ganga/Lib/Localhost/LocalHostExec.py

```diff
@@ -88,14 +88,18 @@
 
 import time #datetime #disabled for python2.2 compatiblity
 
+if os.path.isfile(os.path.abspath(appscriptpath[0])):
```

Comment:
Between your suggestion of adding the WN path to `PATH` and Ulrik's opinion that `File` is ok to use and that the PR should be closed, I'm not sure that putting this code into *this* PR is the best option.